components/about-us/sub-pages/Governance/intro.tsx

```diff
@@ -7,62 +7,91 @@ export default function GovernanceIntroComponent() {
+            mb='1rem'
           >
-            &nbsp;Snapshot&nbsp;
+            Governance is everyone’s responsibility at BanklessDAO
           </Text>
-          votes. Any BANK holder can vote in a snapshot. The more tokens you
-          hold, the more voting power you have. See our recent governance
-          decisions
           <Text
-            color={'red'}
+            color={'white.200'}
             fontSize={{ sm: 'sm', md: 'lg' }}
-            display={'inline'}
+            maxW={{ sm: '30rem', md: '35rem', lg: '50rem' }}
+            my='1rem'
           >
-            &nbsp;here.&nbsp;
-          </Text>
-        </Text>
-        <Box>
-          <Text color={'white.200'} fontSize={{ sm: 'sm', md: 'lg' }}>
-            While decisions are formally ratified on-chain, decision-making
-            happens informally in our Discord channels, our
+            At BanklessDAO, all decisions regarding governance, treasury,
+            projects, and proposals are made through a democratic process. BANK
+            holders ratify decisions on-chain through
+            <Text
+              color={'red'}
+              fontSize={{ sm: 'sm', md: 'lg' }}
+              display={'inline'}
+            >
+              &nbsp;Snapshot&nbsp;
+            </Text>
+            votes. Any BANK holder can vote in a snapshot. The more tokens you
+            hold, the more voting power you have. See our recent governance
+            decisions
             <Text
               color={'red'}
               fontSize={{ sm: 'sm', md: 'lg' }}
-              display={'inline-block'}
+              display={'inline'}
+              maxW={{ sm: '20rem', md: '30rem', lg: '50rem' }}
             >
-              &nbsp;forum posts
+              &nbsp;here.&nbsp;
             </Text>
-            , and our voice rooms.
           </Text>
-        </Box>
+          <Box>
+            <Text
+              color={'white.200'}
+              fontSize={{ sm: 'sm', md: 'lg' }}
+              maxW={{ sm: '30rem', md: '35rem', lg: '50rem' }}
+            >
+              While decisions are formally ratified on-chain, decision-making
+              happens informally in our Discord channels, our
+              <Text
+                color={'red'}
+                fontSize={{ sm: 'sm', md: 'lg' }}
+                display={'inline-block'}
+              >
+                &nbsp;forum posts
+              </Text>
+              , and our voice rooms.
+            </Text>
+          </Box>
+        </Flex>
       </Stack>
+      <Box
+        position='absolute'
+        top={{ base: '0rem', sm: '0rem', md: '20rem' }}
+        left={{ base: '0', sm: '0', md: '2%', lg: '5%', xl: '10%' }}
+        bgGradient='linear(to-br, rgba(255, 16, 19, 0.3) 16.9%, rgba(140, 29, 207, 0.1) 88.02%)'
+        filter='auto'
+        blur='179px'
+        h='55rem'
+        w={{ base: '70%', md: '50rem' }}
+        borderRadius='full'
+        zIndices='hide'
```

Comment:
QUESTION : should this be zIndex?

If not what is zIndices, what does it do here?

Reply:
Thanks for pointing this out, This was a mistake and didn't achieve anything. I intended to use zIndex here but it is pointless, as I am already moving Up ( in the z-axis ) both intro and kickstart section,

Fixed It